xdsl/backend/csl/print_csl.py

```diff
@@ -266,15 +346,36 @@ def descend(self) -> CslPrintContext:
             output=self.output,
             variables=self.variables.copy(),
             _counter=self._counter,
-            _prefix=self._prefix + "  ",
+            _prefix=self._prefix + self._INDENT,
         )
 
 
+def _get_layout_program(ops: BlockOps) -> tuple[csl.CslModuleOp, csl.CslModuleOp]:
```

Comment:
I think it'd be more appropriate to pass the module here instead of the iterable of it's ops?

Reply:
Also, docstrings!